CHANGELOG.md

```diff
@@ -107,29 +110,27 @@ It includes many new rules, including all new techniques introduced in MITRE ATT
 - build: use Python 3.8 for PyInstaller to support consistently running across multiple operating systems including Windows 7 #505 @mr-tz
 - main: correctly match BB-scope matches at file scope #605 @williballenthin
 
-### Changes
+### Breaking Changes
 
 - py3: drop Python 2 support #480 @Ana06
-- deps: bump ruamel yaml parser to 0.17.4 #519 @williballenthin
```

Comment:
removed as I think we should not track these here (related #550)

Reply:
none of the other entries is lost, they're just moved to different sections

sorry, for not making that clearer inititally